app/app.js

```diff
@@ -1,24 +1,24 @@
 const cfg = require('./config');
```

Comment:
this isn't used in this file, I was going to remove it... but I assume you planned to use it for something in the future.

Reply:
That was the plan, however I may have put it in just to ensure .env is read early on in app startup with the intention of hooking up checks for DEBUG and not found a great place to use the cfg variable.